doc/whats-new.rst

```diff
@@ -22,6 +22,11 @@ v2023.10.2 (unreleased)
 New Features
 ~~~~~~~~~~~~
 
+- Added hypothesis strategies for generating :py:class:`xarray.Variable` objects containing arbitrary data, useful for parametrizing downstream tests.
+  Accessible under :py:func:`testing.strategies`, and documented in a new page on testing in the User Guide.
```

Comment:
that's probably not the right role, maybe use `:py:mod:` instead? Not sure though. In doubt, check the generated intersphinx object inventory using either `python -m sphinx.ext.intersphinx` or `sphobjinv`